### DISCUSSION TITLE
I want to show an env var that is a directory but I only want to see the directory name not the whole path

### DISCUSSION THREAD

**Discussion by brunolm** [JanDeDobbeleer/oh-my-posh#3152] [Troubleshoot] [answered]
I want to show an env var that is a directory but I only want to see the directory name not the whole path.

```
{{ .Env.CONDA_PREFIX }}
```

Is there any way to manipulate strings and make it show `foldername` instead of `C:\blah\blah\blah\foldername`?

**Comment by brunolm** [comment, marked as answer]
This kinda did the job

```
{{ regexFind \"\\\\w+$\" .Env.CONDA_PREFIX }}
```

Full command

```
{
  "type": "text",
  "style": "powerline",
  "foreground": "#111111",
  "background": "#c4da04",
  "template": "{{ if eq (regexFind \"\\\\w+$\" .Env.CONDA_PREFIX) \"Anaconda3\" }} {{ else }} \ue235 {{ regexFind \"\\\\w+$\" .Env.CONDA_PREFIX }} {{ end }}"
},
```

**Comment by JanDeDobbeleer** [comment]
@brunolm you can have a look at sprig's [regexFind](http://masterminds.github.io/sprig/strings.html) for this.

I took too long type the answer it seems 😅
